tools/pipelines/build-docs.yml

```diff
@@ -200,12 +207,41 @@ stages:
             script: |
               pnpm i --frozen-lockfile
 
+        - script: |
```

Comment:
Nit: I always recommend adding a brief comment above each task giving a quick explanation as to what is being done and why. In this case, just a quick note that we need to install jq for the subsequent step.

Reply:
This way if other steps are changed in the future, it's easier to tell if this is still needed or not.